modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItDataSchemaSyncTest.java

```diff
@@ -132,7 +131,7 @@ void beforeEach() throws Exception {
      */
     @AfterEach
     void afterEach() throws Exception {
-        IgniteUtils.closeAll(Lists.reverse(clusterNodes));
+        IgniteUtils.closeAll(clusterNodes);
```

Comment:
If you want to keep the reverse, you could use:
```
        Collections.reverse(clusterNodes);
        IgniteUtils.closeAll(clusterNodes);
```

Reply:
reverse - was a kind of hack from past ) Now it`s not an issue any more.